src/modm/platform/i2s/stm32/i2s_base.hpp.in

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2013, Kevin Läufer
+ * Copyright (c) 2013-2017, Niklas Hauser
+ * Copyright (c) 2014, Daniel Krebs
+ * Copyright (c) 2020, Mike Wolfram
```

Comment:
I don't think I worked on this 😜

Reply:
The file is in large parts a copy of a SPI header...